### DISCUSSION TITLE
Sometimes boot hangs after GNOME Display Manager started

### DISCUSSION THREAD

**Discussion by wforumw** [ltsp/ltsp#283] [Q&A]
Hi,
With around 20% of the clients end the boot process after starting the GNOME Display Manager (see picture). No login screen is coming. After schwitching to a  Console, give username/password and run 

> $ startx

 we are logged in and have our Gnome environment.
We have several client and several users, but the issue happens only in around 20% of boot

![gnomeDM](https://user-images.githubusercontent.com/12762849/68384284-96ae2880-0157-11ea-86dc-b784c453e73a.jpeg)

**Comment by alkisg** [comment]
Distribution, version, client graphics card, and contents of ltsp.conf? (hide what you need)

I've seen GDM randomly fail to start on Wayland, but that was a GDM bug, not an LTSP issue.
By switching to VT2 and then back to VT1, GDM showed up after a few seconds.
This was on Debian Buster running in a VM booted from a normal installation, **not** as an LTSP client.
See https://ltsp.github.io/man/ltsp.conf/ for how to disable Wayland for GDM.

**Comment by wforumw** [comment]
Distribution: Debian Buster
graphics card: Intel HD Graphics 610
ltsp.conf (no special configuration, default)

**Comment by alkisg** [comment]
Try `GDM3_CONF=“WaylandEnable=false”` in ltsp.conf like I wrote above.

Also, does it work if you switch to vt2 and then vt1?

If these things work around the problem, then it's a bug in GDM, and you should report it to bugs.debian.org instead of on LTSP.

**Comment by alkisg** [comment]
AFAIK this issue isn't related to the LTSP code; moving to the community.

**Comment by maccadia** [comment]
I just had LigthDM not showing up with MATE. This was cause by a bad / slow connection. Having the server and clients on the same switch really helps.

**Comment by wforumw** [comment]
Thanks,
But we have Server and Clients on the same Gigabit Switch, with LigthDM we don't have the issue but with LigthDM the screensaver does not work properly

**Comment by wforumw** [comment]
I have deactivated Wayland for all clients as it causes many boot problems.
